docs/english/forretningsregler/index.md

```diff
@@ -8,7 +8,15 @@ description: "Validation rules for the VAT tax return"
 
 <table align=center>
 	<tr><th style="width:25%" align=left>Date</th><th align=left> What was changed? </th></tr>
-	    <tr>
+    <tr>
+      <td>2022.05.18</td>
+      <td>
+          <ul>
+            <li> R085 Check for amount of basis exceeds the maximum value according to requirements from DVH</li>
```

Comment:
Også her. "Check the value of the VAT basis does not exceed the allowed maximum value"